go.mod

```diff
@@ -85,3 +85,5 @@ require (
 // If that happens to get released in a 3.X.X version, we can add a constraint to our go.mod
 // for it. If its in 4.X.X, then we need all our transitive deps to upgrade to it.
 replace github.com/dgrijalva/jwt-go => github.com/form3tech-oss/jwt-go v3.2.1+incompatible
+
+replace github.com/ethereum/go-ethereum => github.com/smartcontractkit/go-ethereum v1.9.26-0.20210909013251-729bcfdc948c
```

Comment:
go mod reformatted this from the full commit hash [729bcfdc948c54702783bd7895a0078081388fcd](https://github.com/smartcontractkit/go-ethereum/commit/729bcfdc948c54702783bd7895a0078081388fcd)